Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ruby:alpine
```

Comment:
Could you specify ruby version? I think Ruby 2.5 (latest version) is the best.
Without version, we cannot pin Ruby version from the Dockerfile, so the version will be up unexpectedly.

And probably we should not use alpine. Because RuboCop on Ruby 2.5 with alpine linux may raise SystemStackError. 
See https://bugs.ruby-lang.org/issues/14387 (Japanese), https://github.com/bbatsov/rubocop/issues/5341